test/types/runtime/internal/lifecyle.test-types.ts

```diff
@@ -0,0 +1,35 @@
+// TODO: enable those tests when #7224 got merged
```

Comment:
is this meant to say 7324 to refer to this PR? https://github.com/sveltejs/svelte/pull/7224 does not exist

assuming this did mean 7324, why not just uncomment it now?

Reply:
#7224 is an open PR ("feat: improve types for `createEventDispatcher`"). In order to test the added typesafety features, an upgrade to TypeScript 3.9 is needed.

It is a bit chaotic since both PRs were openend a long time ago ^^.

Reply:
weird. it was giving me a 404 earlier, but seems to be resolving now. the numbers being off by just a single character made me suspicious :smile: 

thanks for clarifying!